spec/double_entry/reporting_spec.rb

```diff
@@ -42,4 +42,146 @@
       end
     end
   end
+
+  describe '::aggregate' do
```

Comment:
I had not seen '::' in describe before. Is there a reason for using this rather than '.'?